keyboards/clack_ee/saf8/config.h

```diff
@@ -0,0 +1,24 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
+
+#pragma once
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x0000
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    clack.ee
+#define PRODUCT         saf8
```

Comment:
At the very least, you need to quote these:
```suggestion
#define MANUFACTURER    "clack.ee"
#define PRODUCT         "saf8"
```
If you don't, this keyboard will stop compiling when [the changes in this branch](https://github.com/vial-kb/vial-qmk/tree/merge-2023-03-12) are merged into the `vial` branch. That merge may or may not happen before your PR is even accepted.

In the future, this information needs to be moved from `config.h` to `info.json`. [If you don't do this, your keyboard will probably stop compiling with the major update after that](https://docs.qmk.fm/#/ChangeLog/20221126?id=usb-ids-refactoring).

That goes for matrix pins and diode direction too. [Please see other examples of how this works in the repository](https://github.com/vial-kb/vial-qmk/blob/47721416a6483a1d759c21a18beae3c0cb7d3eb2/keyboards/jacky_studio/piggy60/info.json).